cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt

```diff
@@ -161,13 +188,37 @@ datasource "ds_get_anomalies" do
 end
 
 datasource "ds_filter_anomalies" do
-  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $param_cost_metric, $param_anomaly_limit
+  run_script $js_filter_anomalies, $ds_get_anomalies, $param_min_spend, rs_org_id, $param_days, $param_dimensions, $ds_rbd_dimensions, $param_cost_metric, $param_anomaly_limit
 end
 
 ###############################################################################
 # Scripts
 ###############################################################################
 
+script "js_rbd_dimensions", type: "javascript" do
+  parameters "param_rbd_dimensions", "ds_rule_based_dimension"
+  result "cleaned_rbd_dimensions"
+  code <<-EOS
+    var cleaned_rbd_dimensions = {
+      rbd_dimensions_ids: [],
+      messages: []
+    };
+    var exist = false;
```

Comment:
Here you're validating if a single object exists, so maybe instead of "exist" you could use "exists".

Reply:
change variable "messagesu" ---> messages